bodhi/tests/server/services/test_releases.py

```diff
@@ -251,6 +261,43 @@ def test_edit_release(self):
         r = self.db.query(Release).filter(Release.name == name).one()
         self.assertEquals(r.state, ReleaseState.current)
 
+    def test_edit_release_critpath_pkgs(self):
+        """
+        Assert that editing critical path packages for a release are saved correctly.
+        """
+        name = u"F22"
+
+        res = self.app.get('/releases/%s' % name, status=200)
+        r = res.json_body
+
+        r["edited"] = name
+        r["csrf_token"] = self.get_csrf_token()
+        r["critpath_pkgs"] = "bodhi"
+
+        res = self.app.post("/releases/", r, status=200)
+
+        r = self.db.query(Release).filter(Release.name == name).one()
+        self.assertEquals(r.critpath_pkgs[0].name, u'bodhi')
```

Comment:
Let's assert that Bodhi is the only critpath package too:

```
self.assertEqual([p.name for p in r.critpath_pkgs], ['bodhi'])
```

Note also that ```assertEquals``` is deprecated in favor of ```assertEqual```.